bin/publish-to-registries.sh

```diff
@@ -38,10 +37,4 @@ if [[ -v CIRCLE_TAG ]]; then
 
   # Push release tags to internal registry
   push_group "${internalTag}" ".${CIRCLE_TAG}"
-
-  # Push latest/no-suffix tags to dockerhub (e.g. heroku/heroku:22)
-  push_group "${publicTag}" ""
-
-  # Push latest/no-suffix tags to internal registry
-  push_group "${internalTag}" ""
```

Comment:
This still needs to be accounted for when pushing inline with buildx

Reply:
So I think one of the issues here, is that Docker doesn't support having multiple architectures represented as one tag locally. So if you retag a local tag with an alias, then push that alias, I don't think it will push the special multi-architecture manifest, only the single architecture of that tag.

Instead, I think use of the `manifest` command will be required:
https://docs.docker.com/engine/reference/commandline/manifest/

See also:
https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/

Reply:
Ah yes, even better :-)

Reply:
Added that (untested in your ci)